tests/test_utils.py

```diff
@@ -677,6 +688,73 @@ def test_psr_utils():
         0.07774704753236616 * u.G,
     )
 
+    # companion mass
+    assert np.isclose(
+        companion_mass(pb, x, inc=90 * u.deg, mpsr=1.4 * u.Msun),
+        0.2906422269961084 * u.Msun,
+    )
+
+    # pulsar mass
+    assert np.isclose(
+        pulsar_mass(pb, x, 0.4 * u.Msun, inc=90 * u.deg), 2.329629042176839 * u.Msun
+    )
+
+
+@given(
+    floats(min_value=0.5, max_value=3),
+    floats(min_value=0.01, max_value=10),
+    floats(min_value=0.04, max_value=1000),
+    floats(min_value=0.1, max_value=90),
+)
+def test_masses(Mpsr, Mc, Pb, incl):
+    """
+    test pulsar and companion mass calculations for a range of values
+    """
+    assume(incl > 0)
+    assume(incl <= 90)
+    assume(Mpsr > 0)
+    assume(Mc > 0)
+    assume(Pb > 0)
```

Comment:
These `assume`s aren't necessary - the `assume` statement exists to reject generated values that are for some reason not acceptable, while you have already ensured that these limits are met in generation.

Reply:
Yes, I wrote these before I put in the full hypothesis limits, so I agree they are redundant.